lib/pf/config.pm

```diff
@@ -321,17 +321,28 @@ Readonly our $IF_INTERNAL => 'internal';
 
 # Interface enforcement techniques
 # connection type constants
-Readonly our $WIRELESS_802_1X   => 0b110000001;
-Readonly our $WIRELESS_MAC_AUTH => 0b100000010;
-Readonly our $WIRED_802_1X      => 0b011000100;
-Readonly our $WIRED_MAC_AUTH    => 0b001001000;
-Readonly our $WIRED_SNMP_TRAPS  => 0b001010000;
-Readonly our $INLINE            => 0b000100000;
-Readonly our $UNKNOWN           => 0b000000000;
+# 1 : Wireless
+# 2 : Eap
+# 3 : Wired
+# 4 : Inline
+# 5 : SNMP
+# 6 : WebAuth
+# 7 : Available
+# 8 : Available
+# 9 : Available
+Readonly our $WIRELESS_802_1X     => 0b110000000;
+Readonly our $WIRELESS_MAC_AUTH   => 0b100000001;
+Readonly our $WIRED_802_1X        => 0b011000010;
+Readonly our $WIRED_MAC_AUTH      => 0b001000011;
+Readonly our $WIRED_SNMP_TRAPS    => 0b001010100;
+Readonly our $INLINE              => 0b000100101;
+Readonly our $UNKNOWN             => 0b000000000;
+Readonly our $WEBAUTH             => 0b000001111;
```

Comment:
Shouldn't we have wired web-auth and wireless web-auth?

Reply:
We need to add a new bit for that